### PR TITLE
feat: delete and archive

### PR DESCRIPTION
**Pull request by Chiragmodi01**
## Delete and Archive

- User can **archive note** from homepage.
- User can **unarchive note** from archive's page.
- User can **delete note** from homepage.
- User can **delete note** from archive's page.
- User can **restore note** from trash page.

## [Live Preview](https://6256f2037742af0008e4be9a--tranquil-choux-2fb728.netlify.app/)

### CODE REVIEW

File: src/assets/index.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
 import notero from './notero.svg';
 import noteroLight from './noteroLight.svg';
+import trashBin from './trashBin.svg';
+import trashBinLight from './trashBinLight.svg';
+import archive from './archive.svg';
+import archiveLight from './archiveLight.svg';
 
-export {notero, noteroLight};
\ No newline at end of file
+export {notero, noteroLight, trashBin, trashBinLight, archive, archiveLight};
```
Comment (aurobindoGupta) [inline src/assets/index.js:8]: nice

File: src/comps/HomeMain.jsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import NoNotes from './NoNotes';
 import NoteCard from './NoteCard';
 
 function HomeMain( ) {
-  const { notesState, notesDispatch, search,  } = useNote()
+  const { notesState, notesDispatch, search, setNote } = useNote()
```
Comment (aurobindoGupta) [inline src/comps/HomeMain.jsx:10]: please use prittier to format code

**Review by harshitpaliwal95 — APPROVED** [review]
code looks good UI Looks so good 
good work Chirag
------
Just Remove comment code

File: src/comps/HomeMain.jsx (hunk 2 above)
Comment (Chiragmodi01) [reply src/comps/HomeMain.jsx:10]: will do at the end, thanks!

**Pull request merged**